src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -200,10 +208,12 @@ class VanillaUmiConsensusCaller(override val readNamePrefix: String,
           }
         }
 
+        val contributions = builder.contributions // NB: cache this value, as it is re-computed each time
```

Comment:
You know, you really don't need to prefix your comments with "NB" - comments _are_ notes to our future selves 😛

Reply:
Some of us have style Tim